### ISSUE TITLE
pyani command is missing

### ISSUE THREAD

**Issue by seoanezonjic** [widdowquinn/pyani#238]
#### Summary:
The pyani command don't work whereas the scripts genbank_get_genomes_by_taxon.py and average_nucleotide_identity.py work perfectly.

#### Description:
I've installed pyani with conda and when i execute the pyani in command line i've got this error:

If 'pyani' is not a typo you can use command-not-found to lookup the package that contains it, like this:
    cnf pyani

The average and get_genomes are found with their respective commands. I need the pyani command becuase I have 7-9 custom genome files in fasta format that I want yo compare con pyani. SO I have to index these files to perform the comparison between the strains and then I want to compare them with the NCBI genomes. If my error is to use the pyani command. What is the correct way to do this analysis (first a custom comparison between my "problem strains" and then, compare them with an specific NCBI taxon)?

#### Reproducible Steps:
Installa last version with conda recipe.

#### Current Output:
If 'pyani' is not a typo you can use command-not-found to lookup the package that contains it, like this:
    cnf pyani

#### Expected Output:
Obtain the help summay when execute
pyani
or
pyani -h

#### pyani Version:
0.2.10

#### installed dependencies
I cannot do this.

#### Python Version:
Python 3.7.3

#### Operating System:
Suse/linux

**Comment by widdowquinn** [comment]
Hi @seoanezonjic - thank you for your interest in `pyani`.

You appear to have installed version 0.2.10, for which `genbank_get_genomes_by_taxon.py` and `average_nucleotide_identity.py` are the appropriate commands. `pyani` as a command works only in v0.3.x, as noted in the documentation at https://github.com/widdowquinn/pyani#documentation

Closed as duplicate of #226 #192 #176 #171 #169 #165

**Comment by seoanezonjic** [comment]
Hi @widdowquinn 
I think  the problem is that the cited documentation seems to lead to the 0.2.X version. If I follow this documentation, I get the 0.2.X version. However, if I perform a github pip installation, the 0.3.X version is installed. I suggest you to check the documentation to see if this is an error or clarify it to avoid this misunderstanding.
Thank you
Pedro Seoane

**Comment by widdowquinn** [comment]
Hi @seoanezonjic,

The two versions of pyani: v.0.2.x and v0.3.x have different command-line APIs and outputs. This is intended behaviour.

You indicated in your initial post that you are using pyani 0.2.10 - so the 0.2.x documentation is the documentation you should be following, to use that version.

You appear instead to have been following the documentation for the development version (v0.3+) in trying to use the `pyani` command.

The [`README.md`](https://github.com/widdowquinn/pyani#documentation) on the `master` branch for `pyani` does say the following:

```
This README.md file provides a quick overview and walkthrough for THE DEVELOPMENT VERSION (v0.3+) OF pyani, and full documentation can be found at the link below:

    pyani v0.3+ Documentation

Older documentation

If you are using an older version of pyani (v0.2.x), then please note that the command-line API has changed, and documentation for this version can be found at the following page:

    pyani v0.2.x Documentation
```

This is a development version and is not yet available through `conda` or `PyPI`.

If, however, you pip install from the GitHub repository `master` branch, you will indeed install v0.3+ and in that case you should follow the version 0.3+ documentation.

In short:

- If you `pip install pyani` from PyPI you will have v0.2.10
- If you `conda install pyani` from conda you will have v0.2.10
- If you `python setup.py install` from a downloaded repository you will have v0.3+
- If you `pip install -e .` (or similar) from a downloaded repository you will have v0.3+

and you should use the documentation for the version you have installed.
